simple_functions.py

```diff
@@ -1,7 +1,9 @@
 # Custom python functions
 
 def double_number(a):
+'''This function takes the input and prints the number before and after doubling it and returns the doubled number'''
     return a+a
 
 def square_number(a):
+'''This function takes the input and prints the original number and the square of it and returns the squared number'''
```

Comment:
Same as above.

Reply:
Fixed the issue 0893bcdeedb6118e88ac9f278b8e9a1b9ab2a5ba